src/components/ProtectedAuthentication.js

```diff
@@ -3,6 +3,7 @@ import { Navigate, Outlet } from 'react-router-dom';
 
 export default class ProtectedAuthentication extends Component {
   render() {
-    return (this.props.isLoggedIn === true) ? <Navigate to='/home'/> : <Outlet/>;
+      return (this.props.isLoggedIn === true) ? <Navigate to='/home'/> : <Outlet/>;
+
```

Comment:
Why do random new lines are introduced in each PR?

Reply:
I believe these lines are introduced while I am resolving the conflicts even though I thought I removed as many extra lines as possible. I'll be more careful next time. Apologies.